include/osquery/numeric_monitoring.h

```diff
@@ -0,0 +1,71 @@
+/**
+ *  Copyright (c) 2014-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under both the Apache 2.0 license (found in the
+ *  LICENSE file in the root directory of this source tree) and the GPLv2 (found
+ *  in the COPYING file in the root directory of this source tree).
+ *  You may select, at your option, one of the above-listed licenses.
+ */
+
+#pragma once
+
+#include <osquery/core.h>
+#include <osquery/plugin.h>
+#include <osquery/query.h>
+
+#include <chrono>
+#include <string>
+
+namespace osquery {
+
+namespace monitoring {
+
+/**
+ * Types for clock and time point in monitoring plugin
+ */
+using Clock = std::chrono::system_clock;
+using TimePoint = Clock::time_point;
+
+using ValueType = double;
```

Comment:
I think it would be better to start with Int and then go to double if we need it

Reply:
Ok, let's make it `long long int`.